eng/pipelines/pr.yml

```diff
@@ -1,5 +1,9 @@
 trigger:
-  - main
+  branches:
+    include:
+      - main
+      - gh-readonly-queue/* # Used to trigger for GitHub merge queues
```

Comment:
don't we want to try this in the typespec-azure repo for a while before moving here?

Reply:
this just allows it to be enabled, it doesn't hurt to have it

Reply:
I am not actually changing to use merge queue in here but that will then just be a lever flip if we decide to